Services/CkDeviceDetector.php

```diff
@@ -16,14 +15,14 @@ class CkDeviceDetector extends DeviceDetector
      */
     public function __construct(string $userAgent = '')
     {
+        parent::__construct($userAgent);
```

Comment:
I think that not calling the parent constructor was on purpose to avoid all unnecessary parsers to be added

Reply:
Yes, you are 50% right. I did it because one of the tests failed. Then I began an analysis and found this method. The IDE said that there must be the call of a parent constructor. After adding it the tests were Ok. I agree with you on refactoring.

Reply:
Ok, could you share your ideas on refactoring?

Reply:
My idea is to delete the call of the parent constructor and add the line after calling the the client parsers: 
$this->addDeviceParser(new Mobile());
so we could define if the device is tablet or mobile.

Reply:
@Alex-Caepio 
Why you add $this->addDeviceParser(new Mobile()); ?